virt/virtual_machines/advanced_vm_management/virt-working-with-resource-quotas.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+[id="virt-working-with-resource-quotas"]
+= Working with resource quotas in {VirtProductName}
+include::_attributes/common-attributes.adoc[]
+:context: virt-working-with-resource-quotas
+
+toc::[]
+
+Resource quotas are implemented in {VirtProductName} similarly to how they are implemented in {product-title}. However, in (VirtProductName}, you are creating and managing resource quotas only for virtual machines.
```

Comment:
I might reword this intro to something simpler, since the 'meat' of this doc change is in the procedure module. You can have an assembly intro that is as simple as "Create and manage resource quotas for virtual machines."

Suggestion: we could also include an xref to the OCP resource quota docs somewhere, if that would be useful.

Reply:
Went with the simple recommendation. I thought about mentioning limits for a more comprehensive abstract sentence but I also don't want to limit the scope of this assembly.